cli/tests/e2e/test_ci.py

```diff
@@ -358,48 +354,84 @@ def automocks(mocker):
         """
     ).lstrip()
 
-    mocker.patch.object(
-        ConfigLoader,
-        "_download_config_from_url",
-        side_effect=lambda url: ConfigFile(None, file_content, url),
-    )
+
+@pytest.fixture(autouse=True)
+def automocks(mocker, mock_ci_api):
+    """
+    Necessary patches to run `semgrep ci` tests
+    """
     mocker.patch.object(
         GitMeta,
         "commit_timestamp",
         FROZEN_ISOTIMESTAMP,
     )
-    mocker.patch.object(
-        ScanHandler,
-        "_get_scan_config_from_app",
-        return_value=out.ScanConfig(
-            deployment_id=DEPLOYMENT_ID,
-            deployment_name="org_name",
-            ignored_files=[],
-            policy_names=["audit", "comment", "block"],
-            rule_config=file_content,
-        ),
+
+
+@pytest.fixture
+def mocked_scan_id() -> int:
+    return 35976
```

Comment:
Can we use a more obviously fake scan_id here? like 12345 ?

Reply:
Yep, good call.

Reply:
yep, done